docs/user-guide/customize.md

```diff
@@ -17,37 +17,40 @@ There's a lot you can do. For example, if you only want to allow:
 
 You can use the [`unit-allowed-list`](../../lib/rules/unit-allowed-list) and [`declaration-property-unit-allowed-list`](../../lib/rules/declaration-property-unit-allowed-list) rules:
 
-```json
+```diff json
 {
-  "rules": {
-    "declaration-property-unit-allowed-list": {
-      "/^border/": ["px"],
-      "/^padding|^gap/": ["rem"]
-    },
-    "unit-allowed-list": ["%", "deg", "px", "rem", "ms"]
-  }
+  "extends": ["stylelint-config-standard"],
++ "rules": {
++   "declaration-property-unit-allowed-list": {
++     "/^border/": ["px"],
++     "/^padding|^gap/": ["rem"]
++   },
++   "unit-allowed-list": ["%", "deg", "px", "rem", "ms"]
++ }
 }
 ```
```

Comment:
[note] Using diffs here and showing `"extends": ["stylelint-config-standard"],` so that this page feels like a continuation of the getting started one.